packages/form/src/test-utils.tsx

```diff
@@ -0,0 +1,17 @@
+// @ts-ignore
```

Comment:
ts ignore should not be necessary. probably something wrong with local setup of the form's pacakge.json

Reply:
fixed